lib/utilities/printable-properties.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var commandProperties = [
+  'name',
+  'description',
+  'aliases',
+  'works',
+  'availableOptions',
+  'anonymousOptions'
+];
+var blueprintProperties = [
+  'name',
+  'description',
+  'availableOptions',
+  'anonymousOptions',
+  'overridden'
+];
+
+function forEachProperty(properties, forEach, context) {
```

Comment:
I think it is a better name for the function you have already defined.

It appears, we will forEach only members with a specific property, hence the naming recommendation of `forEachWithProperty`